lib-rca/src/main/java/com/facebook/openwifi/librca/inputs/RCAParams.java

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifi.librca.inputs;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/** Define root cause analysis configuration parameters */
+public final class RCAParams {
+
+	/** Look-back window in minutes */
+	public final int detectionWindowMin;
+
+	// KPI calculation parameters
+	/** Minimum acceptable estimated throughput (Mbps) */
+	public final double minEstimatedThroughputMbps;
+	/** Percentile (units are %) of estimated throughputs to use as the KPI */
+	public final double throughputAggregationPercentile;
+	/** Maximum acceptable latency (ms) */
+	public final int maxLatencyThresholdMs;
+	/** Maximum acceptable jitter (ms) */
+	public final int maxJitterThresholdMs;
+	/**
+	 * Maximum acceptable disconnection rate (disconnetions per hour). Note that
+	 * this signifies a rate and the units happen to be per hour - this does not
+	 * signify that every contiguous one-hour period be checked.
+	 */
+	public final int maxDisconnectionRatePerHour;
+
+	// High Level metrics parameters
```

Comment:
`// High Level metrics parameters` -> `// High Level metrics threshold` 
Or something like that? They are parameters to compare with

Reply:
Sure, and I can do the same with the low level ones also